### PR TITLE
update sourceComments documentation in README.md

### PR DESCRIPTION
**Pull request by screamingmunch**
expand on sourceComments description in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -351,7 +351,8 @@ var result = sass.renderSync({
   data: 'body{background:blue; a{color:black;}}',
   outputStyle: 'compressed',
   outFile: '/to/my/output.css',
-  sourceMap: true, // or an absolute or relative (to outFile) path
+  sourceMap: true, // or an absolute or relative (to outFile) path,
+  sourceComments: true,
```
Comment (nschonni) [inline README.md]: Not needed since the Sass project doesn't recommend using this option anymore (AFIK)

File: README.md [hunk 2]
```diff
@@ -257,7 +257,7 @@ Used to determine how many digits after the decimal will be allowed. For instanc
 Type: `Boolean`
 Default: `false`
 
-`true` enables additional debugging information in the output file as CSS comments
+`true` enables additional debugging information (line number and source scss file reference, similar to :line_comments in compass) in the output file as CSS comments
```
Comment (nschonni) [inline README.md]: `:line_numbers` is the Ruby Sass one (with `:line_comments`  as an alias) http://sass-lang.com/documentation/file.SASS_REFERENCE.html#options
Compass is depreciated now, so probably shouldn't be mentioned

Comment (xzyfer) [inline README.md]: Yes please remove the reference to compass.

File: README.md [hunk 3]
```diff
@@ -351,7 +351,7 @@ var result = sass.renderSync({
   data: 'body{background:blue; a{color:black;}}',
   outputStyle: 'compressed',
   outFile: '/to/my/output.css',
-  sourceMap: true, // or an absolute or relative (to outFile) path
+  sourceMap: true, // or an absolute or relative (to outFile) path,
```
Comment (xzyfer) [inline README.md]: Please remove the comma at the end of the comment

**Comment by xzyfer** [comment]
Thank you @screamingmunch! ❤️

**Comment by screamingmunch** [comment]
Apologies to all the rookie misses.. first-time contributing to open source.  

Removed references to compass, and removed the comma in code sample(left by previous author).  =)

**Comment by xzyfer** [comment]
No worries @screamingmunch, we're excited you have your help!

**Pull request merged**